CHANGES/5709.misc

```diff
@@ -0,0 +1 @@
+Provide a descriptive error for invalid treeinfo files
```

Comment:
I think it can be .bugfix, because it's a user facing error and we improve it, so it would be good to notify users in the changelog about that